src/js/views/maps/MapWidgetContainerView.js

```diff
@@ -34,6 +35,7 @@ define(["backbone", "models/maps/Map", "views/maps/CesiumWidgetView"], (
       /** @inheritdoc */
       render() {
         this.renderMapWidget(this.el, this.model);
+        this.renderLegendContainer(this.el, this.model);
```

Comment:
Don't need to pass `el` or `model`. Same for mapWidget actually!

Reply:
oh right right!